js/common/lib/env-impl.ts

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {Env} from './env';
+
+export class EnvIml implements Env {
```

Comment:
use a type alias like `type EnvLogLevel = typeof Env['logLevel']` <--- this may not compile, not sure what is the exact syntax, but this is doable